docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -100,6 +100,45 @@ Apart from `extra_links.tmpl` and `extra_tabs.tmpl`, there are other useful temp
 - `body_outer_post.tmpl`, before the bottom `<footer>` element.
 - `footer.tmpl`, right before the end of the `<body>` tag, a good place for additional Javascript.
 
+#### Example: Mermaid.js
+
+If you would like to add [mermaid.js](https://mermaid-js.github.io/) support to Gitea's markdown you simply add:
```

Comment:
FWIW, this link is already 404.

Reply:
Should probably be https://mermaid-js.github.io/mermaid

Reply:
Bloody hell

Reply:
I've sent another PR